torch/_export/non_strict_utils.py

```diff
@@ -446,6 +446,42 @@ def inner(m: torch.nn.Module, prefix_atoms: List[str], constants):
     return constants
 
 
+@contextlib.contextmanager
+def _fakify_module_inputs(
+    args: Tuple[Any],
+    kwargs: Dict[Any, Any],
+    fake_mode: torch._subclasses.fake_tensor.FakeTensorMode,
+):
+    # This context manager is used to fakify module inputs.
+    # Inputs:
+    #   args, kwargs: the args and kwargs containing module inputs that haven't been fakified.
+    #   fake_mode: the fake mode to be used for fakifying script objects. It's the same mode that fakify input tensors.
+
+    ctxs = []
+    for arg in pytree.tree_leaves((args, kwargs)):
+        if isinstance(arg, torch.nn.Module):
+            from torch.export._trace import _fakify_params_buffers
```

Comment:
Hmm should we move this to a util file? `from torch.export._trace import _fakify_params_buffers`